Cucumber/features/project1/pages/patient_page.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'top_part.rb'
+class PatientPage
+
+  include TopPart
+
+  def delete_patient_btn
+    driver.find_element(id: 'org.openmrs.module.coreapps.deletePatient')
+  end
+
+  def delete_reason_field
+    driver.find_element(id: 'delete-reason')
+  end
+
+  def confirm_delete_btn
+    driver.find_element(xpath: "//div[contains(@id,'delete-patient-creation-dialog')]//button[contains(@class,'confirm right')]")
```

Comment:
this seems tricky. Doesn't it find always the first patient and click confirm for it? What happens if there are few patients displayed? Better to pass patient dialog as element or row and then find its confirm button.
Also would look easier in css. 
def confirm_delete_btn(patient_dialog)
  patient_dialog.find_element(css: '.confirm.right')
end